### DISCUSSION TITLE
Where to put the backend configuration

### DISCUSSION THREAD

**Discussion by GD-Dal** [mealie-recipes/mealie#3349] [Q&A]
Hi
I'm testing Mealie in a (unraid) docker, but I'm wondering where to put the backend configuration?
I know I can put them in environment variables in the docker configuration, but surely there must be a better way?
Isn't there a config file somewhere in Mealie I can use instead?

Thanks

**Comment by michael-genson** [comment]
https://docs.mealie.io/documentation/getting-started/installation/installation-checklist/

The only place they can go is the docker environment variables, there's no config file or in-app way to set these.

**Comment by GD-Dal** [comment]
Ok, environment variables it is.
Thanks.
